Lib/typing.py

```diff
@@ -1135,9 +1135,30 @@ def __mro_entries__(self, bases):
         res = []
         if self.__origin__ not in bases:
             res.append(self.__origin__)
+
+        # Check if any base that occurs after us in the bases list is either
```

Comment:
This might make it slightly clearer that we're referring to the local variable `bases` rather than the original`__orig_bases__` tuple:

```suggestion
        # Check if any base that occurs after us in `bases` is either
```